profiles/rockets/tantalus.py

```diff
@@ -41,16 +42,95 @@ def buttons(self):
             "Ping Stage 2": "TANTALUS_STAGE_2_FLARE.PING",
         }
 
+    @property
+    def label_to_dataID(self):
```

Comment:
The non-Tantalus integration tests are failing because this mapping only exists in Tantalus. Could we move the common mappings for both dataIDs and units to a higher level class, then return those along with rocket-specific mappings in these methods?